master/buildbot/process/buildrequest.py

```diff
@@ -23,6 +23,16 @@
 from buildbot.status.buildrequest import BuildRequestStatus
 from buildbot.db import buildrequests
 
+
+class Priority(object):
+    Emergency = 100
+    VeryHigh = 80
+    High =  75
+    TimedScheduler = 70
+    Default = Medium = 50
```

Comment:
I'd make this

Medium = Default = 50 so that all of the "real" names are on the same starting position